### PR TITLE
Added FAQ Page Implementation for Improved User Support

### PR DESCRIPTION
**Pull request by SrijaVuppala295**
# Pull Request Template for GSoC 2024

## Related Issue

[Please list any related issue numbers that this pull request resolves. If none, state "None"]

closes:#287

## Description of Changes

This pull request introduces a new FAQ (Frequently Asked Questions) page aimed at enhancing user support and providing quick answers to common inquiries. The following changes have been made:

- FAQ Component: Created a responsive FAQ component using React, which allows users to expand or collapse questions to view answers.
- Dynamic Content: Integrated a data structure to manage FAQ items dynamically, making it easier to update or add new questions and answers in the future.
- User-Friendly Design: Implemented a clean and intuitive design for the FAQ page, enhancing usability with visual cues such as icons and animations for expanded/collapsed states.
- Navigation Integration: Included navigation functionality to allow users to return to the home page easily, improving overall user experience.
- Styling Updates: Applied CSS styles to ensure that the FAQ page is visually consistent with the rest of the application, including adjustments for mobile responsiveness.
- This FAQ page aims to reduce the volume of support inquiries by providing users with immediate access to commonly sought information, thereby improving user satisfaction and engagement.

## Type of Pull Request

- [ ] Bug Fix
- [x] New Feature
- [ ] Performance Improvement
- [x] Documentation Update
- [x] Code Refactoring
- [ ] Security Enhancement
- [ ] Other (Specify): _______________

(Please check relevant boxes by entering an "X" inside them)

## Screenshots / Demo (if applicable)

image

image

## Checklist for Contributor
use'X' in [ ] to check it .

- [x] I have reviewed my code before submission.
- [x] I have read and followed the project's Contribution Guidelines.
- [x] I have tested the code thoroughly.
- [x] I have attached related issue numbers, screenshots, or videos (if applicable).
- [x] I have added comments to my code where necessary for clarity.
- [x] I have checked the existing code base and my changes do not introduce any breaking changes.
- [x] My PR addresses only one issue; I have created separate PRs for other issues.
- [x] I have ensured that the PR does not include unrelated files.
- [x] I am mindful of the project's coding standards and best practices.

## Additional Information

[Include any additional context or relevant information that may help in reviewing this PR.]

## Contribution Guidelines Reminder

Thank you for contributing to our project! Please ensure you follow the guidelines below:

### Contribution Levels

2. **Task Categories:**
   - **Basic Tasks:** Primarily focused on small changes like minor bug fixes, documentation improvements, and unit testing. Labeled as **Easy**.
   - **Intermediate Tasks:** Moderate-level tasks that involve enhancing existing features, code refactoring, or performance improvements. Labeled as **Medium**.
   - **Advanced Tasks:** Complex tasks like building new features, significant refactoring, or critical bug fixes. Labeled as **Hard**.

3. **Acknowledging Effort:**
   - Exceptional contributions will be recognized with a higher contribution level if substantial effort and innovation are demonstrated.

### Review Process

- PRs will be reviewed by the maintainers, and feedback will be provided. Contributors should address feedback promptly and resubmit the PR if necessary.

- Contributors are encouraged to participate in the review process of other PRs for community growth.

We appreciate your time and effort in improving this project and look forward to your valuable contributions!

### CODE REVIEW

**Comment by SrijaVuppala295** [comment]
@dhairyagothi , please add level 3 label , I have created all the content acc to the UI !!!

**Comment by dhairyagothi** [comment]
image
@SrijaVuppala295 fix this deployment error

**Comment by SrijaVuppala295** [comment]
@dhairyagothi , I have fixed it !! Please review and merge it and please add level 3 label !!

**Comment by dhairyagothi** [comment]
please resolve conflicting files @SrijaVuppala295

**Comment by SrijaVuppala295** [comment]
@dhairyagothi , i  have resolved it , please review it

**Review by dhairyagothi — APPROVED** [review]

File: frontend/src/Pages/hamburger.jsx [hunk 1]
```diff
@@ -155,6 +157,10 @@ const Hamburger = () => {
   const helpClick = () => {
     navigate("/Help");
   };
+  const FaqClick = () => {
+    navigate("/FAQ");
+  };
+
```
Comment (dhairyagothi) [inline frontend/src/Pages/hamburger.jsx:163]: fix the dom link and css it is not opening and overlapping 

image

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by SrijaVuppala295** [comment]
## @dhairyagothi , I have made the changes ,  created a scrollable hamburger menu , created a new PR ==> #505
image

**Pull request closed without merging**